backend/gncitizen/core/taxonomy/routes.py

```diff
@@ -110,6 +281,119 @@ def get_taxon_from_cd_nom(cd_nom):
     """
     """Renvoie la fiche TaxRef de l'espèce d'après le cd_nom"""
     try:
-        return get_specie_from_cd_nom(cd_nom=cd_nom)
+        return get_taxa_by_cd_nom(cd_nom=cd_nom)
     except Exception as e:
         return {"message": str(e)}, 400
+    
+
+@taxo_api.route("/taxonomy/tmedias/types", methods=["GET"])
```

Comment:
C'est donc pas exclusivement géré côté frontend ?

Reply:
ce sont uniquement la définition des routes qui sont coté backend . Ca permet de garder une homogénéité dans la gestion des routes API (notamment pour le renvoi des erreurs) . Par contre , les modifications n'utilisent plus la mise en cache coté backend (ce qui était le cas pour les taxhub_list) . Ce qui a été dit lors de la réunion, proposé et expliqué par Amandine c'était d'avoir une mise en cache des types de médias et attributs coté frontend (et c'est le cas dans la PR , voir le globalCacheService) .